be/src/util/compression/stream_compression.cpp

```diff
@@ -422,6 +426,455 @@ Status ZstandardStreamCompression::decompress(uint8_t* input, size_t input_len,
     return Status::OK();
 }
 
+// lzo
+#if defined(WITH_LZO)
+class LzoStreamCompression : public StreamCompression {
```

Comment:
You'd better add Unit Tests for this newly added method.
It is easy to reuse the existing tests.